app/controllers/clients_controller.rb

```diff
@@ -15,31 +15,23 @@ def edit; end
 
   def create
     @client = Client.new(client_params)
-
-    respond_to do |format|
-      if @client.save
+    if @client.save
+      respond_to do |format|
         format.html { redirect_to @client, notice: 'Client was successfully created' }
```

Comment:
When you have only 1 respond_to i.e. only html, then we dont need to make a respond_to block.
So it can as simple as
```
if @client.save
   redirect_to @client, notice: 'Client was successfully created'
else
  ....